<launch>

  <arg name="rgb_image"
       default="/aeroa_remote/openni_camera/rgb/image_rect_color" />
  <arg name="depth_image"
       default="/aeroa_remote/openni_camera/depth_registered/image_rect" />
  <arg name="hand_pose"
       default="/ssd_hand_detector/people_pose_estimation_2d_hand/pose" />

  <arg name="ksize" value="3" />
  <arg name="extract_joint" value="RHand3" />

  <node name="hand_pose_visualizer"
        pkg="hand_motion_estimator" type="hand_pose_visualizer.py"
        output="screen">
    <remap from="~input/rgb_image" to="$(arg rgb_image)" />
    <remap from="~input/depth_image" to="$(arg depth_image)" />
    <remap from="~input/hand_pose" to="$(arg hand_pose)" />
    <rosparam subst_value="true">
      approximate_sync: true
      ksize: $(arg ksize)
      extract_joint: $(arg extract_joint)
    </rosparam>
  </node>

</launch>
